### PR TITLE
Merge 2.1 into 2.x

### PR DESCRIPTION
**Pull request by mrclay**
@jdalsem @hypeJunction Can you check that I resolved these conflicts correctly?
- [x] mod/notifications/views/default/notifications/subscriptions/collections.php
- [x] mod/reportedcontent/start.php
- [x] mod/thewire/views/default/thewire.js

### CODE REVIEW

File: mod/notifications/views/default/notifications/subscriptions/personal.php [hunk 1]
```diff
@@ -70,7 +70,7 @@
 	'width' => '100%',
 ];
 
-$body = elgg_view_module('info', elgg_echo('notifications:subscriptions:personal:title'));
+$body = elgg_view_module('info', elgg_echo('notifications:subscriptions:personal:title'), '');
```
Comment (mrclay) [inline mod/notifications/views/default/notifications/subscriptions/personal.php]: This is actually a bug fix. A $body is required.

**Comment by mrclay** [comment]
FWIW I manually tested these.

**Comment by hypeJunction** [comment]
Seems right

**Comment by mrclay** [comment]
For reportedcontent, all we did in 2.1 was add library calls to load the lightbox (not necessary in 2.x) and the elgg/reportedcontent module (loaded via menu item deps in 2.x). So this should be OK.

**Pull request merged**